packages/foam-core/src/markdown-provider.ts

```diff
@@ -422,7 +436,14 @@ export function createMarkdownReferences(
         : dropExtension(relativePath);
 
       // [wiki-link-text]: path/to/file.md "Page title"
-      return { label: link.slug, url: pathToNote, title: target.title };
+      return {
+        label:
+          link.rawText.indexOf('[[') > -1
+            ? link.rawText.substring(2, link.rawText.length - 2)
```

Comment:
you raised a good point in your PR comment, I don't know what text goes here exactly, e.g. whether just the target or the raw text, we should look closely at the specs

Reply:
The format we use right now is the correct one if I follow the CommonMark specs, so suggest we stick to this.